addons/account/static/src/components/settings_form_view/res_config_dev_tool.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+import { ResConfigDevTool } from "@web/webclient/settings_form_view/widgets/res_config_dev_tool";
+
+/**
+ * Override of the widget in the settings that handles the "Developer Tools" section.
+ * Provides a button to download XSD files for XML validation.
+ */
+class ResConfigDevToolDownloadXsd extends ResConfigDevTool {
```

Comment:
@antoine162 No export here then?

Reply:
Indeed, export would be necessary only if we wanted to import the class in another file (I think).